### PR TITLE
VideoPress: upload detected .vvt file from videopress block description

### LINKED ISSUE #26261: VideoPress: refresh VideoPress player after uploading chapters

### PR DESCRIPTION
**Pull request by retrofox**
This PR refreshes the player once the new chapters are uploaded via details control component

fixes https://github.com/Automattic/jetpack/issues/26261

#### Changes proposed in this Pull Request:

* VideoPress: upload detected .vvt file from videopress block description

#### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?

#### Testing instructions:

* Edit the chapters list via DetailsControl component
* Save the post
* Confirm the player updates, showing the new chapters 

https://user-images.githubusercontent.com/77539/190680824-624a3ca8-6cf6-44d4-8913-5eef2fe69920.mov

### CODE REVIEW

File: projects/plugins/jetpack/extensions/blocks/videopress/video-chapters/hooks/use-video-item-update/index.js [hunk 1]
```diff
@@ -69,8 +81,30 @@ export function useSyncMedia( { id, title, description } ) {
...
+		// Video Chapters //
+		if ( ! dataToUpdate?.description?.length ) {
+			return;
+		}
+
+		// Upload .vtt file if it description contains chapters
+		const chapters = extractVideoChapters( dataToUpdate.description );
+		if ( ! chapters?.length ) {
+			return;
+		}
```
Comment (dhasilva) [inline projects/plugins/jetpack/extensions/blocks/videopress/video-chapters/hooks/use-video-item-update/index.js]: `generateChaptersFile` already checks this. You could use something like:

```js
const vttFile generateChaptersFile( dataToUpdate.description );
if ( ! vttFile ) return; // vttFile === null
```

**Review by dhasilva — APPROVED** [review]
LGTM :rocket: 
For a follow-up, we should make the chapters icon appear after the first save with chapters :+1:

File: projects/plugins/jetpack/extensions/blocks/videopress/video-chapters/hooks/use-video-item-update/index.js (hunk 1 above)
Comment (retrofox) [reply projects/plugins/jetpack/extensions/blocks/videopress/video-chapters/hooks/use-video-item-update/index.js]: https://github.com/Automattic/jetpack/issues/26268

**Comment by retrofox** [comment]
> LGTM 🚀 For a follow-up, we should make the chapters icon appear after the first save with chapters 👍

https://github.com/Automattic/jetpack/issues/26268

**Comment by retrofox** [comment]
r252721-wpcom

**Pull request merged**